Automattic/pocket-casts-android
# Use landscape images in the Featured cluster

## Description

This adds support for landscape images in the Trending cluster for Engage SDK integration.

## Testing Instructions

1. Apply [this patch](https://github.com/user-attachments/files/17539221/engage.patch).
2. Install fresh version of the app.
3. Open the app
4. Minimize the app.
5. Wait a bit so that the Discover page is crawled.
6. Proceed with the [Engage integration workflow](https://developer.android.com/guide/playcore/engage/workflow) until the 3rd step.
7. Check that the trending cluster uses landscape images.

## Screenshots or Screencast 

![Screenshot_20241028-082408](https://github.com/user-attachments/assets/231a1a3e-739f-410f-835c-dbffcba55d53)

## Checklist
